editor/src/components/editor/editor-component.tsx

```diff
@@ -192,7 +193,7 @@ export const EditorComponentInner = React.memo((props: EditorProps) => {
         editorStoreRef.current.dispatch,
       )
     },
-    [editorStoreRef, namesByKey],
+    [editorStoreRef, namesByKey, setClearKeyboardInteraction],
```

Comment:
the function returned by the hook is not memoized, so it'll always be a new reference, so this will always trip the reference equality check

Reply:
True! I wonder which is nicer, memoizing the hook, or just converting it back to its original form.

Reply:
I fixed it keeping the hook